test/extended/openstack/bz_2073398.go

```diff
@@ -48,20 +51,23 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] Bugfix", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 
 		skipUnlessMachineAPIOperator(ctx, dc, clientSet.CoreV1().Namespaces())
+
+		g.By("preparing the openstack client")
```

Comment:
Since the ProviderClient is cached, that should make no difference any more.

Reply:
But now that you mention it, maybe there's a way to leverage a `BeforaAll` to generate a ProviderClient, instead of using my homebrewed caching

Reply:
@mandre  I just found this neat explanation from Ginkgo's author on why it's better to reduce common initialisation at a minimum: https://github.com/onsi/ginkgo/issues/70#issuecomment-42882976

Reply:
Looks like the author changed his mind because there is now `BeforeAll()` in ginkgo. Are you saying that client initialization should go in `BeforeSuite()`?

Reply:
I tried that, but there's one already in the vendored Origin code, and you can't have two.

Reply:
Here it is! The cached ProviderClient is back, and everything seems to work. PTAL!

Reply:
(building a ServiceClient from a ProviderClient is essentially free)